governator-core/src/main/java/com/netflix/governator/PropertiesPropertySource.java

```diff
@@ -4,21 +4,35 @@
 
 import javax.inject.Singleton;
 
+import com.google.inject.Binder;
 import com.google.inject.Module;
 import com.google.inject.Provides;
 import com.netflix.governator.spi.PropertySource;
 
-public class PropertiesPropertySource extends AbstractPropertySource {
+public final class PropertiesPropertySource extends AbstractPropertySource implements Module {
```

Comment:
Gak.  Properties is legacy Java with all synchronized methods.  Why do we want this in 2016, and not a Map<String,String>?